cmd/set/domain/domain.go

```diff
@@ -52,9 +52,16 @@ func NewdomainCmd() *cobra.Command {
 			if !strings.Contains(p.Callbackurl, opts.Domain) {
 				return &cmdutil.FlagError{Err: errors.New("Entered Domain Not Found")}
 			}
+			if !cmdutil.DomainValidation.MatchString(opts.DomainMod)  {
+				return &cmdutil.FlagError{Err: errors.New("Invalid Domain")}
+			}
+			
 			if opts.Domain == "http://localhost" || opts.Domain == "http://127.0.0.1" {
 				return &cmdutil.FlagError{Err: errors.New("Cannot Update Default Domains")}
 			}
```

Comment:
We can remove this validation as now we can add `http://localhost` and `http://127.0.0.1` to the Web Apps,